src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -98,51 +107,39 @@ public boolean isEmpty() {
         return size == 0;
     }
 
-    private void checkIndexIfInBound(T value, int index) {
+    private void checkIndexIfInBound(int index) {
         if (index >= 0 && index < size) {
```

Comment:
it's a void method, and we don't actually need return here. how can we achieve it?